target/scripts/check-for-changes.sh

```diff
@@ -143,7 +140,7 @@ do
       chown -R 5000:5000 /var/mail
     fi
 
-    _changedetector_notify 'debug' 'Restarting services due to detected changes..'
+    _changedetector_notify 'debug' 'Restarting services due to detected changes'
```

Comment:
There are a couple other logs in this script that also end with `..`, did you want to drop those too? They were all probably by me as a way to convey intent of logging an action that was about to be performed (_but an ellipsis for such is apparently `...` which looks weird to me_).

Reply:
I have made a decision here: I'd avoid `..` or `...` at the end and indicate that

1. something is about to start by writing a gerund / -ing at the end of the verb (e.g. "Starting to extract certificates")
2. something has finished by writing (simple) past (e.g. "Extracted certificates")

I adjusted `check-for-changes.sh` now, and we can continuously adjust the other scripts.

Reply:
Done in ee3152f346241dd0b726571a38edbad15e8fe6e8